src/Analyzer.Utilities/CSharp/ITypeSymbolExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Analyzer.Utilities
+{
+    public static class ITypeSymbolExtensions
+    {
+        public static bool IsAssignableTo(this ITypeSymbol fromSymbol, ITypeSymbol toSymbol, Compilation compilation)
```

Comment:
We should **really** request this API to be added to the compiler layer. Ideally, `Conversion` and its public APIs would be exposed at the core layer. There is realistically no language specific code that you are adding here.